src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -12,466 +14,121 @@ namespace System.Tests
     public class Perf_Double
     {
         public static readonly double dValue = 1.23456789E+5;
-        private const int iterCount = 20000;
+        private string _string;
```

Comment:
Technically for this to have the impact you want it to have, I think it needs to be volatile (but I could be wrong).